src/Log/Log.php

```diff
@@ -214,7 +214,9 @@ protected static function _loadConfig(): void
      */
     public static function reset(): void
     {
-        static::$_registry = null;
+        if (!empty(static::$_registry)) {
```

Comment:
can we use strict null checks for object init check?

Reply:
If I do strict null check then phpstan complains the the check is redundant since the property's docblock doesn't say it's nullable. If I add `|null` for the property both phpstan and psalm start complaining that usage like `static::$_registry->foo` being potentially null access :slightly_smiling_face: